recipes/aws-c-s3/all/conanfile.py

```diff
@@ -69,10 +69,13 @@ def package(self):
     def package_info(self):
         self.cpp_info.filenames["cmake_find_package"] = "aws-c-s3"
         self.cpp_info.filenames["cmake_find_package_multi"] = "aws-c-s3"
+        self.cpp_info.set_property("cmake_file_name", "aws-c-s3")
         self.cpp_info.names["cmake_find_package"] = "AWS"
         self.cpp_info.names["cmake_find_package_multi"] = "AWS"
+        self.cpp_info.set_property("cmake_target_name", "AWS")
         self.cpp_info.components["aws-c-s3-lib"].names["cmake_find_package"] = "aws-c-s3"
         self.cpp_info.components["aws-c-s3-lib"].names["cmake_find_package_multi"] = "aws-c-s3"
+        self.cpp_info.components["aws-c-s3-lib"].set_property("cmake_target_name", "aws-c-s3")
```

Comment:
```suggestion
        self.cpp_info.components["aws-c-s3-lib"].set_property("cmake_target_name", "AWS::aws-c-s3")
```
I _think_ these targets must be absolute now, right ?

Reply:
same as on the other PR. this requires conan 1.43.0 which is not yet deployed on CCI, so feel free to ignore these suggestions